projects/application/DatSphere-en.md

```diff
@@ -0,0 +1,62 @@
+# Project Name DatSphere
+
+## Category 
+application
+
+## Project Description
+An open-source, decentralized drive service built on ipfs, filecoin
+
+## Project Status
+under development
+### Done
+- Account Module  
+    - Login
+    - Register
+    - Password Change
+    - Reset Password
+- Drive Module
+    - navigate files
+    - upload file/folder
+    - create and update text-file/folder
+    - download file/folder
+- IPFS Module
+    - import remote ipfs hash
+    - generate ipfs hash for uploaded files
+### Todo
+- integrate FileCoin
+
+## Previews
+![Image](https://drive.google.com/file/d/1rICFF2y_pn3eH7_g-_P8WoSIjN3aqPfx/view)
```

Comment:
Images are not showing up. Please fix it or add images in `asset` folder

Reply:
fixed